index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Alumni+Sans+Pinstripe&family=Poppins&family=Roboto+Flex:opsz,wght@8..144,500&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+    <nav id="toolbar" class="nav-bar">
+        <span id="logo">BBL</span>
+        <img src="assets/images/Union.svg" id="menu" alt="menu button">
+    </nav>
+</header>
+    <br><br>
+  
+    <section id="headline" class="head_section"> 
+       <div id="greet"> <h1>I’m Brian, Glad to see  you!</h1> </div>
+       <br>
+        <p>I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <p id="connect">LET’S CONNECT</p>
```

Comment:
- [ ] Please kindly note that it is [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) to use an indentation of 2 spaces for readability. kindly change your indentations, please.

Reply:
I have fixed the indentation to 2 spaces.